### PR TITLE
feat(blog): adding IoT blog post

### PR DESCRIPTION
**Pull request by amyshwang**
Adding new iot article and replace the blog landing page tag.

Remove time-series and add iot:
Before:
Screenshot 2023-07-06 at 09 56 03
Now:
Screenshot 2023-07-06 at 09 54 30

### CODE REVIEW

File: blog/2023-07-06-visualizing-iot-data-questdb.mdx [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+<Screenshot
+  alt="Diagram showing how sensor data is sent to Mosquitto, Telegraf, QuestDB, and then Grafana."
+  height={591}
+  src="/img/blog/2023-07-06/overview.webp"
+  width={770}
+  title="An overview of the tools used to process the IoT data."
+/>
+
+Since we don't have our own sensor to collect the data and send it directly to
```
Comment (javier) [inline blog/2023-07-06-visualizing-iot-data-questdb.mdx:48]: sensor or sensors (plural) here? Not sure

File: blog/2023-07-06-visualizing-iot-data-questdb.mdx [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+Upon finishing the tutorial, you may want to clean up any dangling resources,
+such as the tutorial network we created. Removing a network requires the removal
+of existing resources attached to that first. To remove the containers and the
+network used during this tutorial, run the following commands:
+
+```bash
+docker ps -qa --filter="network=tutorial" | xargs -n1 docker kill
+docker network rm tutorial
+```
+
```
Comment (javier) [inline blog/2023-07-06-visualizing-iot-data-questdb.mdx:330]: If we are cleaning up resources, I'd also suggest removing the images, as those take about 1Gb of disk. The command would be

`docker docker rmi telegraf eclipse-mosquitto grafana/grafana questdb/questdb`

**Review by javier — CHANGES_REQUESTED** [review]
If we adding a clean section, we should properly clean up and remove the docker images. which take quite a bit of space

File: blog/2023-07-06-visualizing-iot-data-questdb.mdx (hunk 2 above)
Comment (amyshwang) [reply blog/2023-07-06-visualizing-iot-data-questdb.mdx:330]: Do you mean:
```
docker rmi telegraf eclipse-mosquitto grafana/grafana questdb/questdb
```
Also I guess this is on top of the existing commands?

File: blog/2023-07-06-visualizing-iot-data-questdb.mdx (hunk 1 above)
Comment (amyshwang) [reply blog/2023-07-06-visualizing-iot-data-questdb.mdx]: Changed to plural.

File: blog/2023-07-06-visualizing-iot-data-questdb.mdx (hunk 2 above)
Comment (javier) [reply blog/2023-07-06-visualizing-iot-data-questdb.mdx:331]: You are correct on both things. This command (with a single docker word at the beginning) should be executed after the docker network rm, just as you wrote it :)

**Review by javier — APPROVED** [review]

**Pull request merged**